test/unit/core.js

```diff
@@ -1532,6 +1532,38 @@ testIframe(
 	}
 );
 
+testIframe(
```

Comment:
I see you'd like to run these tests in the browser as well as node. But, it'd be ideal if the files required for these tests were only built if core tests were run.

Reply:
The Node ones test the Node usage, the browser ones test that bundlers work correctly. Bundlers generate files for browsers so I need a browser here.

Reply:
Yeah, I realized after that it still needs to build in Node, but it *could* only do that if the `core` module is being tested.

Reply:
These tests no longer exists since I moved them to Node + JSDOM.